js/src/ui/MethodDecoding/methodDecoding.css

```diff
@@ -16,6 +16,7 @@
 */
 
 .method {
+  font-family: 'Roboto Mono' !important;
```

Comment:
Should be `font-family: …, monospace;` to tell the computer that you want monospace event if loading the font fails.